src-docs/src/views/empty_prompt/_types_of_use_cases.tsx

```diff
@@ -218,7 +224,10 @@ export const typesOfUseCases: {
     example: {
       iconLight: pageNotFoundLight,
       iconDark: pageNotFoundDark,
+      iconLight2x: pageNotFoundLight2x,
+      iconDark2x: pageNotFoundDark2x,
       title: <h2>Page not found</h2>,
+      alt: 'Astronaut flying in a planet',
```

Comment:
```suggestion
      alt: 'An outer space illustration. In the background is a large moon and two planets. In the foreground is an astronaut floating in space and the numbers '404'.,
```

After reading the discussion in https://github.com/elastic/eui/pull/5969#discussion_r923880716 I just wanted to add more description here. Feel free to solicit help from docs writers if you want!